README.md

```diff
@@ -18,44 +18,69 @@ When you submit a pull request, a CLA-bot will automatically determine whether y
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
-## AutoRest extension configuration
+## Autorest plugin configuration
+
+Please don't edit this section unless you're re-configuring how the Swift extension plugs in to AutoRest. AutoRest needs the below config to pick this up as a plug-in - see https://github.com/Azure/autorest/blob/master/docs/developer/architecture/AutoRest-extension.md
+
+**Swift code gen**
 
 ``` yaml
+version: 3.0.6267
 use-extension:
-  "@microsoft.azure/autorest.modeler": "2.1.22"
+  "@autorest/modelerfour": "4.15.410"
+
+modelerfour:
+  # this runs a pre-namer step to clean up names
+  prenamer: true
+
+  # this will flatten models marked with 'x-ms-client-flatten'
+  flatten-models: true
+
+  # TODO: What does this do?
+  flatten-payloads: false
```

Comment:
This flattens request body types, I believe.

https://github.com/Azure/autorest.modelerfour/blame/master/modelerfour/readme.md#L132